mk/spdk.common.mk

```diff
@@ -48,6 +48,7 @@ OS := $(shell uname)
 COMMON_CFLAGS = -g $(C_OPT) -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wmissing-declarations -fno-strict-aliasing -march=native -m64 -I$(SPDK_ROOT_DIR)/include
 
 COMMON_CFLAGS += -include $(SPDK_ROOT_DIR)/config.h
+COMMON_CFLAGS += -I$(SPDK_ROOT_DIR)/lib/env_dpdk
```

Comment:
I don't think this should be necessary; nothing should be including files from lib/env_dpdk directly, since all of the public API should be declared in include/spdk/env.h.